typescript/cli/src/rebalancer/utils/tryFn.ts

```diff
@@ -0,0 +1,12 @@
+import { monitorLogger } from './logger.js';
+
+export async function tryFn(fn: () => Promise<void>, context: string) {
+  try {
+    await fn();
+  } catch (error) {
+    monitorLogger.error(
+      { context, err: error as Error },
+      `Error in ${context}`,
+    );
```

Comment:
should this rethrow considering it's used in a nested way so the error bubbles up?